cmd/catchupsrv/main.go

```diff
@@ -18,11 +18,13 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"io/ioutil"
 	"net/http"
+	"path"
 	"strconv"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
For consistency with other code in this repo, can you use `github.com/algorand/go-algorand/logging`?

Reply:
Done.